src/raft_log.rs

```diff
@@ -1402,4 +1489,7 @@ mod test {
             }
         }
     }
+
+    #[test]
+    fn test_maybe_persist() {}
```

Comment:
What's the test content?

Reply:
I remove these lines. The test is `test_maybe_persist_with_snap` above.